Update docs to clarify the dns configmap format. (#29988)
kubernetes/website: content/en/docs/tasks/administer-cluster/nodelocaldns.md

Once enabled, node-local-dns Pods will run in the kube-system namespace on each of the cluster nodes. This Pod runs [CoreDNS](https://github.com/coredns/coredns) in cache mode, so all CoreDNS metrics exposed by the different plugins will be available on a per-node basis.

You can disable this feature by removing the DaemonSet, using `kubectl delete -f <manifest>` . You should also revert any changes you made to the kubelet configuration.

## StubDomains and Upstream server Configuration

> StubDomains and Upstream servers specified in the `kube-dns` configmap in the kube-system namespace will be automatically picked up by node-local-dns pods. The configmap contents need to follow the format shown [here](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#example-1). The `node-local-dns` configmap can also be modified directly with the stubDomain configuration in the Corefile format. Some cloud-providers might not allow modifying `node-local-dns` configmap directly. In those cases, `kube-dns` configmap can be updated.

Reviewer: ```suggestion
StubDomains and upstream servers specified in the `kube-dns` ConfigMap in the `kube-system` namespace
are automatically picked up by `node-local-dns` pods. The ConfigMap contents need to follow the format
shown in [the example](/docs/tasks/administer-cluster/dns-custom-nameservers/#example-1).
The `node-local-dns` ConfigMap can also be modified directly with the stubDomain configuration
in the Corefile format. Some cloud providers might not allow modifying `node-local-dns` ConfigMap directly.
In those cases, the `kube-dns` ConfigMap can be updated.
```

Reviewer: Please 1) wrap the long lines when appropriate; 2) use ConfigMap rather than configmap because 'configmap' is not an English word.

Author: Thanks! This is very useful.
